subprojects/internal-performance-testing/internal-performance-testing.gradle.kts

```diff
@@ -65,7 +65,7 @@ dependencies {
     implementation(library("jsch"))
     implementation(library("commons_math"))
     implementation(library("jcl_to_slf4j"))
-    implementation("org.openjdk.jmc:flightrecorder:7.0.0-SNAPSHOT")
+    implementation("org.gradle.org.openjdk.jmc:flightrecorder:7.0.0-alpha01")
```

Comment:
I think we can also remove the OSS repository, can't we?

Reply:
Good point. Done.